selenium-testing/src/main/java/com/testng/selenium/v1/App.java

```diff
@@ -0,0 +1,13 @@
+package com.testng.selenium.v1;
+
+/**
+ * Hello world!
+ *
+ */
+public class App 
```

Comment:
Is this class required?

Reply:
Done: https://github.com/bcgov/ecc-ofm/pull/319/commits/e4c38d2ffd83f1fc0cd1e233099a8f907635a9ac